src/plots/rectanglePlot.ts

```diff
@@ -251,6 +253,50 @@ export module Plots {
       });
     }
 
+    /**
+     * Gets the Accessor for label.
+     */
+    public label(): AccessorScaleBinding<any, string>;
+    /**
+     * Sets label to the result of an Accessor.
+     *
+     * @param {Accessor<string>} label
+     * @returns {Plots.Rectangle} The calling Rectangle Plot.
+     */
+    public label(label: Accessor<string>): Plots.Rectangle<X, Y>;
+    public label(label?: Accessor<string>): any {
+      if (label == null) {
+        return this._propertyBindings.get(Rectangle._LABEL_KEY);
+      }
+
+      this._bindProperty(Rectangle._LABEL_KEY, label, null);
```

Comment:
Since the label-drawing code is independent from the main part of the drawing, does it even make sense to bind this as a property? In particular, there's no `Scale`, so there's not really a point in storing the binding so we can compute the extent or something.